### PR TITLE
Add About Route

### PR DESCRIPTION
**Pull request by burkehollandmsft**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -7,4 +7,9 @@ router.get('/', function (req, res, next) {
   res.render('index', { products: products });
 });
 
+/* GET home page. */
+router.get('/about', function (req, res, next) {
```
Comment (burkeholland) [inline routes/index.js:11]: Why do we need this route?

File: views/about.jade [hunk 2]
```diff
@@ -0,0 +1,8 @@
+extends layout
+
+block content
+  #app
+    header
+      .container.section
+        h1.is-size-1 About Page
+        a(href="/") Go Back
```
Comment (burkeholland) [inline views/about.jade:8]: Does this always works?

File: routes/index.js (hunk 1 above)
Comment (burkehollandmsft) [reply routes/index.js:11]: Because it's a website

File: views/about.jade (hunk 2 above)
Comment (burkehollandmsft) [reply views/about.jade:8]: sometimes it works 100% of the time

**Pull request merged**